server/form-pages/apply/required-referrals/accommodation-referral-details/dtrDetails.ts

```diff
@@ -41,6 +71,9 @@ export default class DtrDetails implements TasklistPage {
     return {
       [this.questions.reference]: this.body.reference,
       [this.questions.date]: DateFormats.isoDateToUIDate(this.body.date),
+      [this.questions.localAuthority]: this.body.localAuthorityAreaName
+        ? this.body.localAuthorityAreaName
+        : 'No local authority selected',
```

Comment:
is this the desired response if getLocalAuthorities errors?

Reply:
Yeah, It's not ideal but for the benefit of the assessor, this is probably enough. Unfortunately, CAS3 no longer has a content person to bounce this off of 😬